.travis.yml

```diff
@@ -32,8 +32,9 @@ install:
 script:
 - make install
 - go test ./pkg/...
-- go vet ./...
 - go test ./test/e2e/...
+- go vet ./...
+- ./hack/check_license.sh
```

Comment:
I think we should do format check before any tests. In that way, we fail fast on any format issue.  maybe move  `go vet ./...` and `./hack/check_license.sh` above line 34?

Reply:
Travis runs every line in the `script` section, even if one of the early ones fails: https://docs.travis-ci.com/user/customizing-the-build/#customizing-the-build-step

Reply:
Do you think we should put it in `before_script`? That way it should fail before doing any of the other tests.

Reply:
`Travis runs every line in the script section, even if one of the early ones fails:` Good find.
I don't really want to add fmt check to `before_script` script. This might be confusing what needs to be in `before_scrip`t and in `script`. I think we should just keep it as it it now util travis add the fail fast feature. https://github.com/travis-ci/travis-ci/issues/1066. Not sure what's going with travis, this feature has been requested for past 5 years. lol.